Dockerfile

```diff
@@ -109,6 +109,9 @@ RUN pip install scipy
 RUN pip install git+https://github.com/chanzuckerberg/srst2
 # TODO: Test both pip installations, consider keeping pip use consistent
 RUN pip3 install pandas
+# Bedtools for obtaining total reads for each gene and calculating reads per million
+RUN wget https://github.com/arq5x/bedtools2/releases/download/v2.28.0/bedtools
```

Comment:
Have you tried `apt-get install bedtools` ?
Always prefer packages to this source code directly download.